src/Uno.Templates/content/unoapp/.template.config/template.json

```
@@ -881,14 +906,22 @@
       "value": "((useDependencyInjection && useHttp) || (server && useRecommendedAppTemplate))"
     },
     "useUnitTests": {
-      "type": "computed",
+      "type": "generated",
+      "generator": "regexMatch",
       "dataType": "bool",
-      "value": "(preset == 'recommended' &&  tests == unit)"
+      "parameters": {
+      "source": "testsEvaluator",
+      "pattern": ".*unit.*"
```

```suggestion
      "pattern": "(unit)"
```
Wouldn't this fix the issue?

---

nope :-)